lib/krane/kubernetes_resource/daemon_set.rb

```diff
@@ -3,6 +3,7 @@
 module Krane
   class DaemonSet < PodSetBase
     TIMEOUT = 5.minutes
+    SYNC_DEPENDENCIES = %w(Pod)
```

Comment:
We're intentionally not capturing nodes here?

Reply:
Yep, Katrina left a comment about it above but TLDR we only ever fetch it once anyway apparently so we've explicitly skipped it.